src/style.css

```
@@ -338,7 +338,7 @@ h2 {
     margin-bottom: 0;
 }
 
-.hide-furigana rt, .display-none, .hide-emojis .inquery-emoji {
+.hide-furigana rt, .display-none, .hide-emojis .inquery-emoji, .hide-streak {
```

Can remove this if hiding "streak" instead of "streak-container" per my other comment
```suggestion
.hide-furigana rt, .display-none, .hide-emojis .inquery-emoji {
```